cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/scopes/Scope.kt

```diff
@@ -210,6 +213,33 @@ sealed class Scope(
                 }
         }
 
+        // If the symbol was still not resolved, and we are performing an unqualified resolution, we
+        // search in the
+        // languages builtin scope for the symbol
+        if (list.isNullOrEmpty() && !qualifiedLookup) {
+            list = mutableListOf()
+            // If the language has builtins we can search there for the symbol
+            (languageOnly as? HasBuiltins)?.let {
+                val builtinNamespace = it.builtinsNamespace
```

Comment:
Naming is a little bit inconsistent. It should probably be builtin*s* all the time.

Reply:
Agreed